### ISSUE TITLE
Can´t choose Organization for Customer

### ISSUE THREAD

**Issue by Pupp3tm4st3r** [zammad/zammad#211]
Hi,

I created a new organization and customer - when editing the customer I can´t choose the organization. When pasting the organization name und saving the changes, it doesn´t write them - so when I open the edit mask for the customer again, the organization is not saved. 

By now I´m using the docker image.

Best regards
Simon

**Comment by thorsteneckel** [comment]
Hi @Pupp3tm4st3r - thanks for your feedback. I can reproduce the issue in my docker container, too. Let us take a look. We will come back to this shortly.

**Comment by rolfschmidt** [comment]
I also can reproduce the problem on the docker image. It's not reproduceable on a source installation via ubuntu.

**Comment by thorsteneckel** [comment]
Hi @Pupp3tm4st3r - we fixed the issue with the latest Docker image. It was caused by an old Postgresql version (8.4) for CentOS 6. We updated it to the latest version (9.6) which solved the issue for us.

We found the following lines in the production log which showed the cause:
`Error while reserving job: PG::FeatureNotSupported: ERROR:  SELECT FOR UPDATE/SHARE is not allowed in subqueries`

Docker images are updated. Thanks to @rolfschmidt and @rkaldung for the help finding and fixing this.
